### PR TITLE
test: add test with escaped characters

### PR DESCRIPTION
**Pull request by evilebottnawi**
**What kind of change does this PR introduce?**
Tests

**Did you add tests for your changes?**

it is tests

**If relevant, did you update the README?**

not required

**Summary**

Ref: https://github.com/webpack-contrib/css-loader/issues/459

**Does this PR introduce a breaking change?**

no

**Other information**

not required

### CODE REVIEW

File: test/simpleTest.js [hunk 1]
```diff
@@ -18,18 +18,29 @@ describe("simple", function() {
 	test("simple2", ".class { a: b c d; }\n.two {}", [
 		[1, ".class { a: b c d; }\n.two {}", ""]
 	]);
+	test("escape characters (uppercase)", ".class { content: \"\\F10C\" }", [
+		[1, ".class { content: \"\\F10C\" }", ""]
+	]);
+	// Need uncomment after resolve https://github.com/css-modules/postcss-modules-local-by-default/issues/108
```
Comment (michael-ciniawsky) [inline test/simpleTest.js]: Not sure about these tbh, maybe better keep a note and add them when it is resolved :)

Comment (evilebottnawi) [inline test/simpleTest.js]: @michael-ciniawsky i think best note is not in code, some issue can resolve in a long time :sob: Be good merge this as is and if some submit about problem related to this add link to issue in other repo maybe it is help solve the problem faster

**Comment by michael-ciniawsky** [comment]
I will update the test suite today, so we can test within a 'real world' scenario (`ETWP` && `style-loader`)

**Comment by evilebottnawi** [comment]
@michael-ciniawsky If I'm not mistaken in `postcss` `v6` we have builin `value-parse` maybe we can migrate to this, `postcss` more stable than other tools for this :smile: Now i think we can leave it at that.

**Comment by michael-ciniawsky** [comment]
Instead of `css-selector-tokenizer`, I wanted to try taht ou but hadn't the time to do so until now, if `postcss-value-parser || postcss v6` could replace it I'm all in

**Comment by evilebottnawi** [comment]
@michael-ciniawsky yep, I generally see for the first time the project using this module, we tried it in other projects, it did not meet expectations :sob: let's wait `postcss` `v6`, btw we can migrate on `postcss-value-parser` (or `postcss-values-parser`) it is more stable

**Comment by michael-ciniawsky** [comment]
👍  I will open an issue for general overview with all available options here soon, so we can then triage this, but very good catch I'm surprised to be honest 😛

**Pull request merged**